### ISSUE TITLE
Remaining Mobile Mockups

### ISSUE THREAD

**Issue by nazmicancalik** [bounswe/bounswe2018group9#82]
According to the refined requirements, we are missing some of the android mockup pages. 
These include:

- [x] Message Screen
- [x] Search Functionality

I have started implementing these mockup pages. I will check the requirements again and then create pages that are missing. After I complete, if there is something missing please inform me.

**Comment by nazmicancalik** [comment]
I will update this comment as I create the remaining pages/functionalities.
Following pages/functionalities are added:
* Guest Landing Page (Appears when clicked `Not Right Now` in sign up page.)

**Comment by nazmicancalik** [comment]
Guys, I have added the remaining android mockup pages.They are located under android-mockups branch.
The requirements above message are satisfied. 

Here are the pages I have added:
* [Create Event Page](https://github.com/bounswe/bounswe2018group9/blob/android-mockups/wiki/images/create_event_android_new.png)
* [Search Page](https://github.com/bounswe/bounswe2018group9/blob/android-mockups/wiki/images/search_android.png)
* [Guest Trending Page](https://github.com/bounswe/bounswe2018group9/blob/android-mockups/wiki/images/guest_trending_android.png)
* [Send Message](https://github.com/bounswe/bounswe2018group9/blob/android-mockups/wiki/images/send_message_android.png)

Ps: Since my work is done in android mockups. I would like to volunteer for preparation of `Class Diagrams`. I can help @akaogluozgur @byklyci 

@gumityolcu In case there is an issue on android mockups, I will not close this issue. Please write your reviews

**Comment by nazmicancalik** [comment]
@gumityolcu  Umit  thanks for your reviews and heads up, I really appreciate that. 
* Adding the price to the event creation page was in my mind and I forgot. :D
* Also, our sign in page is already prepared. Its just that we decided according to our scenario with Ceyda, we decided not to include the sign in page in mockups, however we should put that you are right. 
* Also we should really create a timeline page where users can see their own events as you said. 

Yes, lets head to whatsapp :D

**Comment by okyksl** [comment]
@nazmicancalik great work! I overviewed the mockups, and would like to add following points for @gumityolcu:

* **1.1.7.2.** Should we need to show _location based search more explicitly_?
* **1.1.6.5.** If I'm not missing, shouldn't we also need to add annotations section to [Create Event Page](https://github.com/bounswe/bounswe2018group9/blob/android-mockups/wiki/images/create_event_android_new.png)?

**Comment by gumityolcu** [comment]
So, I finished the work and added the functionalities I wrote in my first review.

While I worked I noticed some other missing mockups that weren't obvious while reading the requirements. These are:

1- We need 2 seperate pages for Event Details, 1 for Guest users who can't share or post photos. another for members.

2- We need 2 settings pages for verified and unverified users.

3- The events shown in Guest Trending page should be the same as the events in Member Trending page

4- We need 3 profile pages, 1 for verified users, 1 for unverified users who choose to make their profile public, 1 for profiles that are not public.

I added these to the repo in the android-mockups branch.

@okyksl You are right, I didn't think of that but right now @dkilyas is waiting for my feedback on the web mockups so I have to work on that. When I finish I will try to add the Maps and Annotations to the mockups.

**Comment by nazmicancalik** [comment]
@gumityolcu I checked the mockups, this is great work :+1:  @gumityolcu If you want I can add maps and annotations too

**Comment by gumityolcu** [comment]
@nazmicancalik That would be great! I should review web mockups as soon as possible so ilyas can work on them before the weekend 😄

**Comment by okyksl** [comment]
@nazmicancalik @gumityolcu I believe it would be overkill to design every single detail/feature that meticulously. For example,

> 2- We need 2 settings pages for verified and unverified users.

how about we just add a verified button to the settings page? Or if you wish, I think we can omit this detail. 

The mockup details we should focus on are about the _information flow specific to our application_. **Annotations and location based search** are main components related with that so we should design those mockups with explicit details. However, if we go deeper and deeper as @gumityolcu's comments, you guys would have a lot of extra work just for _conveying small details_ of the application (I'm certain that if we search, we can find more and more missing mockups based on the crazy requirements we have).

So to sum it up, you guys did one hell of a job! Don't spend extra energy on small and less important details but on main components and features. You know that we can always use your extra energy somewhere else in the project :)

**Comment by nazmicancalik** [comment]
@okyksl You are totaly right, I will do focus more on map and annotations part of the mobile mockups. Other than that I think we have reached to the elaboration detail we want already. I think we should as a team more focus on Class and Sequence Diagrams because of the emphasis instructors made.

**Comment by nazmicancalik** [comment]
Guys I added google maps link to the event creation page so that people can add location from Google Maps. And also now they can add annotations. New version of the mockup (create-event-last) is [here](https://github.com/bounswe/bounswe2018group9/blob/android-mockups/wiki/images/android-createevent-last.png)

Please give feedback, otherwise I am assuming the android mockups are done. And I think that they give pretty much detail for a mockup, and we should more focus on Diagrams.
